pynamodb/models.py

```diff
@@ -349,12 +349,12 @@ def update_item(self, attribute, value=None, action=None, conditional_operator=N
         if len(expected_values):
             kwargs.update(expected=self._build_expected_values(expected_values, UPDATE_FILTER_OPERATOR_MAP))
         kwargs[pythonic(ATTR_UPDATES)] = {
-            attribute: {
+            attribute_cls.attr_name: {
```

Comment:
ya just so we don't have to keep accessing it that way.. seems error prone if someone changes this in the future, but may just be a matter of opinion